app/Http/Controllers/Administration/UserController.php

```diff
@@ -27,7 +27,7 @@ class UserController extends Controller
 	 */
 	public function list(): Collection
 	{
-		return User::query()->where('id', '>', 0)->get();
+		return User::query()->where('id', '>', 0)->get(); // @phpstan-ignore-line
```

Comment:
Add a comment what it is that we are ignoring?

Reply:
Fixed in https://github.com/LycheeOrg/Lychee/pull/1336/commits/3817459b78f214d434e1cf8b6cb86820b4631eae.